File: src/Entity/Reservation.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

use Symfony\Component\Validator\Constraints as Assert;

/**
 * Reservation
 *
 * @ORM\Table(name="reservation", indexes={@ORM\Index(name="reservation_ibfk_1", columns={"id_stade"})})
 * @ORM\Entity(repositoryClass="App\Repository\ResevartionRepository")
 */
class Reservation
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_reservation", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idReservation;

    /**
     * @var int
     *
     * @ORM\Column(name="heure", type="integer", nullable=false)
     */
    private $heure;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=225, nullable=false)
     */
    private $nom;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var \Stade
     *
     * @ORM\ManyToOne(targetEntity="Stade")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_stade", referencedColumnName="id_stade")
     * })
     */
    private $idStade;

    public function getIdReservation(): ?int
    {
        return $this->idReservation;
    }

    public function getHeure(): ?int
    {
        return $this->heure;
    }

    public function setHeure(int $heure): self
    {
        $this->heure = $heure;

        return $this;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getIdStade(): ?Stade
    {
        return $this->idStade;
    }

    public function setIdStade(?Stade $idStade): self
    {
        $this->idStade = $idStade;

        return $this;
    }


}

File: src/com/mycompany/myapp/entities/Reservation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author asus
 */
public class Reservation {
    private int id_reservation ;
    private int id_stade;
    private int heure;
    private String nom;
    private String date;
    

    public Reservation(int id_reservation, int id_stade, int heure) {
        this.id_reservation = id_reservation;
        this.id_stade = id_stade;
        this.heure = heure;
    }

    public Reservation(int id_reservation, int id_stade, int heure, String nom) {
        this.id_reservation = id_reservation;
        this.id_stade = id_stade;
        this.heure = heure;
        this.nom = nom;
    }

    public Reservation(int id_stade, int heure, String nom) {
        this.id_stade = id_stade;
        this.heure = heure;
        this.nom = nom;
    }

    public Reservation(int id_reservation, int id_stade, int heure, String nom, String date) {
        this.id_reservation = id_reservation;
        this.id_stade = id_stade;
        this.heure = heure;
        this.nom = nom;
        this.date = date;
    }

    public Reservation(int id_stade, int heure, String nom, String date) {
        this.id_stade = id_stade;
        this.heure = heure;
        this.nom = nom;
        this.date = date;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }
    

    public Reservation(int id_stade, int heure) {
        this.id_stade = id_stade;
        this.heure = heure;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }


    public int getId_reservation() {
        return id_reservation;
    }

    public void setId_reservation(int id_reservation) {
        this.id_reservation = id_reservation;
    }

    public int getId_stade() {
        return id_stade;
    }

    public void setId_stade(int id_stade) {
        this.id_stade = id_stade;
    }

    public int getHeure() {
        return heure;
    }

    public void setHeure(int heure) {
        this.heure = heure;
    }

    @Override
    public String toString() {
        return "Reservation{" + "id_reservation=" + id_reservation + ", id_stade=" + id_stade + ", heure=" + heure + '}';
    }
    
}
